languages/dart/keywords/abstract.md

```diff
@@ -0,0 +1,47 @@
+# Abstract
+
+The keyword `abstract` is used to define an *abstract class* - a class that can not be instantiate directly, which means
+you can not create an object from it. Conversely, a class that can be instantiated directly is called a *concrete class*.
+
+Abstract types can define interfaces which hide the implementation details and showing only functionality to the user. 
```

Comment:
Suggestion: Abstract types are interfaces that hide the implementation details and show only functionality to the user.